lib/litani.py

```diff
@@ -67,6 +68,47 @@ class TimeoutExpired(Exception):
     pass
 
 
+@dataclasses.dataclass
+class CacheDir:
+    cache_dir_path: pathlib.Path = None
+
+    def get(self, lookup_path=os.getcwd()):
```

Comment:
Both this method and the `cache_dir_path` field should be static. The whole point is that we shouldn't need to create a `CacheDir` object every time we want to retrieve the path, since that will entail re-traversing the directory tree. We only want this method called once during the entire run of the program.